### PR TITLE
Duracloud: don't read entire chunks into memory

### PR DESCRIPTION
**Pull request by mistydemeo**
Fixes #9155.

### CODE REVIEW

File: storage_service/locations/models/duracloud.py [hunk 1]
```diff
@@ -260,6 +260,22 @@ def move_to_storage_service(self, src_path, dest_path, dest_space):
                 dest = entry.replace(src_path, dest_path, 1)
                 self._download_file(url, dest)
 
+    def _process_chunk(self, f, chunk_path):
+        bytes_read = 0
+        bytes_to_read = 1024 * 1024  # 1MB
+
+        with open(chunk_path) as fchunk:
+            while bytes_read < self.CHUNK_SIZE:
+                bytes = f.read(bytes_to_read)
```
Comment (Hwesta) [inline storage_service/locations/models/duracloud.py]: `bytes` is builtin...

File: storage_service/locations/models/duracloud.py [hunk 2]
```diff
@@ -260,6 +260,22 @@ def move_to_storage_service(self, src_path, dest_path, dest_space):
                 dest = entry.replace(src_path, dest_path, 1)
                 self._download_file(url, dest)
 
+    def _process_chunk(self, f, chunk_path):
+        bytes_read = 0
+        bytes_to_read = 1024 * 1024  # 1MB
+
+        with open(chunk_path) as fchunk:
```
Comment (Hwesta) [inline storage_service/locations/models/duracloud.py]: `open` defaults to `'r'`, and this should be `'w'`

**Comment by mistydemeo** [comment]
All tests are passing!

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**